docs/organizer-guide/confs/website.rst

```diff
@@ -0,0 +1,293 @@
+  The schedule should be in the ``_data/<shortcode>-<year>-schedule.yaml`` file.
+* At some point the conference sells out. ``flagticketsonsale`` is switched off,
+  ``flagsoldout`` is switched on.
+* During the conference, if live streaming is available, ``flaglivestreaming``
+  is switched on. It is switched off after the conference ends.
+* After the conference ends, ``flagpostconf`` is switched on and remains on.
+* When videos are published on the Write the Docs website, ``flagvideos``
+  is switched on and remains on.
+
+Each time the state is updated, check the top/bottom buttons to ensure they
+still fit. For example, once tickets have sold out, the button "buy a ticket"
+should be removed.
+
+
+General conference config file
+------------------------------
+
+If you're setting up a new conference, the easiest may be to take a YAML
+config file from the previous conference, and adjust it.
+
+The items in the general conference config file are:
+
+* ``name``: the dynamic part of the name of the conference, e.g. ``Portland``
+  for Write the Docs Portland.
+* ``shortcode``: the shortcode as used in file paths, config names, and other
+  references to the conference, together with the year. Currently used are
+  ``portland``, ``prague``, and ``australia``.
+* ``year``: the conference year, e.g. ``2020``.
+* ``city``: the city for the conference.
+* ``local_area``: the state or country for the conference.
+* ``area``: the wider area for the conference. Currently used are
+  ``North America``, ``Europe``, and ``Australia``.
+* ``area_adj``: the adjective form of the area, e.g. ``Australian``.
+* ``tz``: the timezone the conference is located in.
+* ``email``: general contact email for the conference.
+* ``color``: the accent color for conference branding, e.g. ``blue`` for 2020.
+* ``photos.default``: the default header image.
+* ``buttons.top`` and ``buttons.bottom``: the buttons highlighted on the top
+  and bottom of the conference pages. Each button has two keys: ``text`` for
+  the button text and ``link`` or ``link_absolute`` for a site-relative or
+  an absolute link for the button. These tend to change as the conference
+  goes through different stages, e.g. a button to submit a talk or watch the
+  live stream. Top and bottom are often identical. Example::
+
+    buttons:
+      top:
+         - text: Sponsor the conference
+           link: /sponsors/prospectus
+         - text: Submit a Talk
+           link: /cfp
+      bottom:
+         - text: Sponsor the conference
+           link: /sponsors/prospectus
+         - text: Submit a Talk
+           link: /cfp
+
+* ``tickets``: the conference ticket prices. The keys are fixed and referred
+  to in other places. Example::
+
+    tickets:
+      corporate:
+        price: $500
+      independent:
+        price: $275
+      student:
+        price: $100
+
+* ``sponsorship``: the conference sponsorship prices. The keys are fixed and
```

Comment:
what does "fixed" mean? the amounts for tix/sponsorships change between conferences and years sometimes, but it's something that needs to be decided by consensus.